src/components/CustomizedPipeline/CustomizePipelines.tsx

```diff
@@ -174,6 +175,7 @@ const Row: React.FC<{
                     variant={ButtonVariant.secondary}
                     href={prURL}
                     showIcon
+                    isDisabled={!prURL}
```

Comment:
Not sure how I feel about a disabled external link. Maybe we should have a tooltip explaining why it is disabled?

Reply:
Ideally the pac state would not be pending when URL isn't available and we shouldn't see a disabled/empty link, but I had disabled it just to be sure.

Replaced it with a fallback repository link now, since the link component doesn't support tooltips by default.
But I can add a tooltip here if that would be better.